client/src/utils/helpers.ts

```diff
@@ -3,3 +3,16 @@ export const capitalize = (string?: string): string =>
 
 export const getWeatherSymbol = (symbol?: string): string =>
   `https://developer.foreca.com/static/images/symbols/${symbol}.png`;
+
+export const throttle = (callback: any, limit: number) => {
```

Comment:
why do you need this function?

Reply:
I've followed article https://medium.com/@jrcreencia/persisting-redux-state-to-local-storage-f81eb0b90e7e where it said that `saveState` should not be called too often "as it uses the expensive `JSON.stringify` operation", and it was suggested to use this utility function called `throttle` from `lodash` library, but I did not want to include library just for one function. So I found this simple function as a standalone.

Maybe it is redundant in this case, because `saveState` is called when feedback state changes, and this state is not updated that often, well, because you can't fill out forms that quickly :D But I think it would be nice to use this to persist `themeType`, and that could change often, as it only requires user to toggle switch-button.

Reply:
ok, got your idea :) looks good, on real project you can also use throttle function imported from lodash library